cassis/typesystem.py

```diff
@@ -349,6 +349,8 @@ def descendants(self) -> Iterator["Type"]:
 
 
 class TypeSystem:
+    NAMESPACE_SEPARATOR = "."
```

Comment:
Why is this here?

Reply:
Because it is there in the Apache UIMA Java SDK.

Reply:
But you seem to not use it

Reply:
One of the constants defined in the Cas reference it.